platforms/android/src/org/crosswalk/engine/XWalkWebViewEngine.java

```diff
@@ -101,13 +101,18 @@ public void onNewIntent(Intent intent) {
                     @Override
                     public Object onMessage(String id, Object data) {
                         if (id.equals("captureXWalkBitmap")) {
-                            XWalkWebViewEngine.this.webView.captureBitmapAsync(
-                                    new XWalkGetBitmapCallback() {
-                                @Override
-                                public void onFinishGetBitmap(Bitmap bitmap,
-                                        int response) {
-                                    pluginManager.postMessage(
-                                            "onGotXWalkBitmap", bitmap);
+                            // Capture bitmap on UI thread.
+                            XWalkWebViewEngine.this.cordova.getActivity().runOnUiThread(new Runnable() {
```

Comment:
Could we make `runOnUiThread()` as a common function? like [this](https://github.com/fujunwei/cordova-crosswalk-engine/blob/capture/platforms/android/src/org/crosswalk/engine/XWalkWebViewEngine.java#L172), then its reusable.